Dockerfile

```diff
@@ -94,7 +96,7 @@ COPY root /
 
 RUN chmod +x /usr/local/bin/run.sh /usr/local/bin/occ /etc/periodic/15min/nextcloud
 
-VOLUME ["/data"]
+VOLUME ["/data","/opt/nextcloud"]
```

Comment:
Oops, that change should not have been pushed, you were right to reject it. Sorry about that. I ended up commenting that out in the end because I found that as you said, it clears it on restart. I ended up enabling the LetsEncrypt setup through configuration in my docker-compose.yml file, which is probably better anyway. Thanks again.